src/qt/walletmodel.cpp

```diff
@@ -37,9 +37,9 @@ WalletModel::WalletModel(std::unique_ptr<interfaces::Wallet> wallet, interfaces:
     QObject(parent), m_wallet(std::move(wallet)), m_node(node), optionsModel(_optionsModel), clientModel(_clientModel), addressTableModel(nullptr),
     transactionTableModel(nullptr),
     recentRequestsTableModel(nullptr),
-    cachedEncryptionStatus(Unencrypted),
-    cachedNumBlocks(0)
+    cachedEncryptionStatus(Unencrypted)
 {
+    cachedLastBalanceUpdate = uint256();
```

Comment:
In commit "[WalletModel] Balance and Txrecord status change polling check based on the block hash and not the height." (997129bdb45c01390bffce991b680d58c9e7d18b)

Suggest dropping initialization in the constructor body and using member initialization